### DISCUSSION TITLE
Question on the Sampling Method of OptunaSearchCV

### DISCUSSION THREAD

**Discussion by YSKRE** [optuna/optuna#4686] [General]
I am using OptunaSearchCV for cross-validated hyperparameter optimization of various machine learning models (Xgboost, Catboost, LightGBM, KNN, etc.). When using Optuna, I was able to select various samplers like TPEsampler, QMCSampler, etc. However, I could not find any related content in the documentation of OptunaSearchCV, which led me to ask these questions:

**1. What sampler is used in the hyperparameter optimization process in OptunaSearchCV? (Is it the TPESampler, which is the default setting of Optuna?)**

**2. Is it currently not possible to select the sampler to be used in OptunaSearchCV?**

**Comment by nzw0301** [comment]
1. When `study` argument is `None` of `OptunaSearchCV`, it uses the default sampler: `TPESampler`
2. You can specify the sampler by creating `study`.

**Reply by YSKRE** [reply]
Thank you for your response!!
